tests/network/expose.go

```diff
@@ -60,10 +60,14 @@ var _ = SIGDescribe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:c
 
 	type ipFamily string
 	const (
-		ipv4            ipFamily = "ipv4"
-		ipv6            ipFamily = "ipv6"
-		dualIPv4Primary ipFamily = "ipv4,ipv6"
-		dualIPv6Primary ipFamily = "ipv6,ipv4"
+		ipv4                     ipFamily = "ipv4"
+		ipv6                     ipFamily = "ipv6"
+		dualIPv4Primary          ipFamily = "ipv4,ipv6"
+		dualIPv6Primary          ipFamily = "ipv6,ipv4"
+		Ipv4Description                   = "over default IPv4 IP family"
```

Comment:
I don't think SonarCloud is smarter than us in this. Moving descriptions into a constant does not make the code any more bulletproof, it only makes it difficult for us to read it. In the case of Ginkgo descriptions, I would vote to ignore the "String literals should not be duplicated" warning.

Reply:
OK. I removed this entire commit.